packages/toolkit/config/paths.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
 	srcDir: './assets/',
 	cssLoaderPaths: ['./assets/css', './includes/blocks'],
 	copyAssetsDir: './assets/',
+	blocksDir: './includes/blocks/',
```

Comment:
This is currently designed for usage in a `10up-theme`. But it can be changed on a per-project basis via the path config